### PR TITLE
Fix more qml warnings

### PR DESCRIPTION
**Pull request by FlexW**

### CODE REVIEW

File: src/gui/tray/Window.qml [hunk 1]
```diff
@@ -531,27 +533,8 @@ Window {
...
-                                        anchors.fill: parent
-                                        anchors.margins: 1
-                                        color: appEntry.hovered ? Style.lightHover : "transparent"
-                                    }
-                                    
-                                    Accessible.role: Accessible.PopupMenu
-                                    Accessible.name: qsTr("Apps menu")
-                                }
-
-                                Accessible.role: Accessible.MenuItem
```
Comment (mgallien) [inline src/gui/tray/Window.qml:552]: why do you remove the accessibility attached properties ?
I am not sure they are forbidden here ?

**Comment by camilasan** [comment]
These are warnings from Qt 5.15. I think we can fix these warnings once we upgrade.

**Comment by mgallien** [comment]
> These are warnings from Qt 5.15. I think we can fix these warnings once we upgrade.

I have asked @FlexW to bump the requirement for Qt to 5.15 to comply with that

File: src/gui/tray/Window.qml (hunk 1 above)
Comment (FlexW) [reply src/gui/tray/Window.qml:552]: That was a mistake. Thanks.

File: src/gui/tray/Window.qml [hunk 2]
```diff
@@ -511,14 +511,16 @@ Window {
...
                         y: (trayWindowAppsButton.y + trayWindowAppsButton.height + 2)
-                        readonly property Item listContentItem: contentItem.contentItem
-                        width: Math.min(listContentItem.childrenRect.width + 4, Style.trayWindowWidth / 2)
-                        height: Math.min(implicitHeight, maxMenuHeight)
                         closePolicy: Menu.CloseOnPressOutsideParent | Menu.CloseOnEscape
-
-                        background: Rectangle {
-                            border.color: Style.menuBorder
-                            radius: 2
+                        width: {
```
[the remark this answers is not in the document]
Comment (FlexW) [reply src/gui/tray/Window.qml:515]: Otherwise, it's not possible to get the width adjusted to the content.

**Pull request closed without merging**